src/UserProfile/ReduxUserProfile.js

```diff
@@ -30,48 +30,53 @@ export const fetchAccess = () =>
       .then(msg => dispatch(msg));
 
 
-const requestDeleteKey = accessKey => ({
+const requestDeleteKey = (jti, exp) => ({
   type: 'REQUEST_DELETE_KEY',
-  access_key: accessKey,
+  jti,
+  exp,
 });
 
 const clearDeleteSession = () => ({
   type: 'CLEAR_DELETE_KEY_SESSION',
 });
 
 
-const deleteKey = (accessKey, keypairsApi) =>
+const deleteKey = (jti, exp, keypairsApi) =>
```

Comment:
Add a comment explaining what 'jti' and 'exp' are?
@param {string} jti is the token id
@param {number} exp is expiration?  - why is this needed?

Reply:
`exp` is required by `blacklist` API